annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastProcessor.java

```diff
@@ -32,8 +42,16 @@ public synchronized void init(ProcessingEnvironment processingEnv) {
 
     @Override
     public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+        if (roundEnv.processingOver()) {
+            try {
+                addServiceEntry(SERVICE_TYPENAME, SERVICE_PROVIDER,
+                    nameClassMap.values().stream().toArray(Element[]::new));
+            } catch (IOException e) {
+                processingEnv.getMessager().printMessage(Diagnostic.Kind.ERROR, "Unable to update Creator service");
+            }
+        }
         if (annotations.contains(fastTypeElement)) {
-            HashMap<String, Element> nameClassMap = collectFastTypes(roundEnv);
+            nameClassMap = collectFastTypes(roundEnv);
```

Comment:
Actually once we've generated CreatorImpl we can't add new types to it, so we should output an error if new annotated types appear in the next round